### ISSUE TITLE
Use github Projects to track this?

### ISSUE THREAD

**Issue by cboettig** [ropensci/devroadmap#3]

**Comment by sckott** [comment]
thanks for the issue Carl, what do you mean exactly by "track this"? - do you mean each package or higher level topic gets a card, then move it through the stages?

**Comment by cboettig** [comment]
Yes, exactly - was thinking columns would be stages, and a card for each package (or each package release).  


Stages might reflect something like the "6 week cycles" discussed on Slack the other day; e.g.  in this 6-week (or maybe 3-mo) session, here's the packages on deck; all others maybe on the roadmap but only for another cycle.  An individual 6-week cycle might focus on a common theme or themes, e.g. 'biodiversity', or 'bug-fix-only', etc.  Just brainstorming here, not trying to dictate.  @richfitz probably has some better ideas.

Might be worth talking to @jimhester and other tidyverse folks about how they pace development cycles too? https://www.tidyverse.org/articles/2017/08/contributing/
